packages/passport-interface/src/RequestTypes.ts

```diff
@@ -269,9 +269,9 @@ export interface ListSingleFeedRequest {
  * some PCDs, given the particular feed and credential that the
  * user supplies.
  */
-export interface PollFeedRequest<T extends PCDPackage = PCDPackage> {
+export interface PollFeedRequest {
   feedId: string;
-  pcd?: SerializedPCD<PCDOf<T>>;
+  pcd: SerializedPCD<SemaphoreSignaturePCD>;
```

Comment:
This seems incorrect - different feeds can have different credentials, right?

Reply:
Previously we accepted `SemaphoreSignaturePCD` or `EmailPCD`. Now we accept only `SemaphoreSignaturePCD`, and if you want to include an `EmailPCD` you have to include it in the `FeedCredentialPayload` (alongside the timestamp).

Reply:
To elaborate a bit: my interpretation of the issue was that we can't use a plain EmailPCD as a credential because it can be re-used, and that it should be replaced by a SemaphoreSignaturePCD that encodes the EmailPCD in the `signedMessage`.

This means that the only credential that will appear in a request is a SemaphoreSignaturePCD, either as a standalone credential or as a wrapper around an EmailPCD. This value cannot have any other type.

Reply:
Currently in practice yes, you're right. However, I don't think that should always be the case. Eg i can imagine another feed that requires you to submit an eddsa signature using a private key that the user has. That feed could still require watermarking, etc. 

tl;dr we aren't building SemaphorePass with SemaphoreFeeds and semaphore credentials, we're building Zupass with PCD feeds and PCD credentials.

Reply:
Will return this to being generic